### PR TITLE
Add as_string visitor for Unknown node

### LINKED ISSUE #1264: 'AsStringVisitor' object has no attribute 'visit_unknown'
```python
>>> import astroid
>>> astroid.nodes.Unknown().as_string()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tusharsadhwani/code/marvin-python/venv/lib/python3.9/site-packages/astroid/nodes/node_ng.py", line 609, in as_string
    return AsStringVisitor()(self)
  File "/Users/tusharsadhwani/code/marvin-python/venv/lib/python3.9/site-packages/astroid/nodes/as_string.py", line 56, in __call__
    return node.accept(self).replace(DOC_NEWLINE, "\n")
  File "/Users/tusharsadhwani/code/marvin-python/venv/lib/python3.9/site-packages/astroid/nodes/node_ng.py", line 220, in accept
    func = getattr(visitor, "visit_" + self.__class__.__name__.lower())
AttributeError: 'AsStringVisitor' object has no attribute 'visit_unknown'
>>> 
```
### `python -c "from astroid import __pkginfo__; print(__pkginfo__.version)"` output

2.8.6-dev0

### PR DESCRIPTION
**Pull request by cdce8p**
## Description

This PR adds an `as_string` visitor method for the `Unknown` node, similar to [`visit_uninferable`](https://github.com/cdce8p/astroid/blob/40d556070fd566b7bb43cb3ed0e498045b88f2d0/astroid/nodes/as_string.py#L637-L638).

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Related Issue

Closes #1264

### CODE REVIEW

**Review by Pierre-Sassoulas — APPROVED** [review]

File: ChangeLog [hunk 1]
```diff
@@ -6,8 +6,12 @@ What's New in astroid 2.9.0?
 ============================
 Release date: TBA
 
+
 * Always treat ``__class_getitem__`` as a classmethod.
 
+* Add missing ``as_string`` visitor method for ``Unknown`` node.
+
+  Closes #1264
```
Comment (cdce8p) [inline ChangeLog:14]:
```suggestion
* Always treat ``__class_getitem__`` as a classmethod.

* Add missing ``as_string`` visitor method for ``Unknown`` node.

  Closes #1264

```

**Comment by cdce8p** [comment]
@Pierre-Sassoulas Please wait a minute. I think I need to fix something first 😞

**Comment by cdce8p** [comment]
@Pierre-Sassoulas I wanted to fix a small typo in my merge commit, so I used the forbidden force push to main. Didn't think that you would do anything else that quickly. I'll see if I can apply the commit manually.

**Comment by Pierre-Sassoulas** [comment]
Sorry ! Let me know when you want to release 2.9 :)

**Comment by cdce8p** [comment]
> Sorry ! Let me know when you want to release 2.9 :)

Think I got it now. I did the merge locally and pushed the commit to `main`.

**Comment by cdce8p** [comment]
@Pierre-Sassoulas Don't know if you fetched before my force push. Just check that your local branch is up-to date with upstream.

**Comment by tusharsadhwani** [comment]
I love that you guys are so quick at working on astroid. I never see such quick work of issues in other projects. Keep it up.

**Comment by cdce8p** [comment]
Thanks @tusharsadhwani ❤️ We'll try our best.

**Comment by Pierre-Sassoulas** [comment]
> I never see such quick work of issues 

Thanks, but it's not always that fast, your MR was opened at the "right time" 😄

**Pull request merged**